#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_man.files.typo.dpatch by A. Costa <agcosta@gis.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: manpage typo

@DPATCH@
--- hdparm/hdparm.8.old	2005-11-21 23:40:00.000000000 +0000
+++ hdparm/hdparm.8	2005-11-21 23:40:18.000000000 +0000
@@ -206,7 +206,7 @@
 will unlock the door mechanism.  Normally, Linux
 maintains the door locking mechanism automatically, depending on drive usage
 (locked whenever a filesystem is mounted).  But on system shutdown, this can
-be a nuisance if the root partition is on a removeable disk, since the root
+be a nuisance if the root partition is on a removable disk, since the root
 partition is left mounted (read-only) after shutdown.  So, by using this
 command to unlock the door
 .B after
